contracts/ValidatorSet/ValidatorSetBase.sol

```diff
@@ -16,11 +16,11 @@ abstract contract ValidatorSetBase is IValidatorSet, Initializable {
 
     uint256 public currentEpochId;
 
+    uint256 public currentValidators;
```

Comment:
currentValidators must represent the active validators count, but it is decreased on ban only

Reply:
Since the count will be only for active validators, I adjusted the logic.